tasks/test_jest.js

```diff
@@ -1,18 +1,16 @@
 const platform = require('os').platform();
-const config = require('./utils/ui_framework_test_config');
 
 module.exports = function (grunt) {
-  grunt.registerTask('uiFramework:test', function () {
+  grunt.registerTask('test:jest', function () {
     const done = this.async();
-    Promise.all([uiFrameworkTest()]).then(done);
+    runJest().then(done);
   });
 
-  function uiFrameworkTest() {
+  function runJest() {
     const serverCmd = {
-      cmd: /^win/.test(platform) ? '.\\node_modules\\.bin\\jest.cmd' : './node_modules/.bin/jest',
+      cmd: 'node',
       args: [
-        '--env=jsdom',
-        `--config=${JSON.stringify(config)}`,
+        /^win/.test(platform) ? '.\\scripts\\jest' : './scripts/jest'
```

Comment:
I suggest the `resolve` function from `path` to handle this, but I also don't think node complains about forward slashes in ports on windows